runtime/core/speaker/speaker_engine.cc

```diff
@@ -67,48 +67,33 @@ void SpeakerEngine::ApplyMean(std::vector<std::vector<float>>* feat,
 
 void SpeakerEngine::ExtractFeature(const int16_t* data, int data_size,
     std::vector<std::vector<std::vector<float>>>* chunks_feat) {
-  // NOTE(cdliang): extract feature with chunk by chunk
   if (data != nullptr) {
```

Comment:
why extract feature chunk by chunk here instead of extract at once?

Reply:
The performance difference between the two is not significant, and `chunk by chunk` is more commonly used. The code has been refactored and still extract feature chunk by chunk.